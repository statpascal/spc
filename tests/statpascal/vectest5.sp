program test6;

function q (a: int64vector): int64vector;
    begin
	q := combine (a + 1, a + 2, 100, 101, 102, 103, 104, 105, 106, 107, 108, 109);
        writeln (size (a))
    end;

procedure p;
    var a: vector of int64;
begin
    a := 17;
    writeln (q (combine (a, a, a)))
end;

begin p end.