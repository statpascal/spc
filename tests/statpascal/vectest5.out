3
18 18 18 19 19 19 100 101 102 103 104 105 106 107 108 109 
